src/bundle/Resources/views/content/tab/content.html.twig

```diff
@@ -1,5 +1,21 @@
+<section class="ez-fieldgroup ">
+    <h5 class="ez-fieldgroup-name" style="overflow: hidden">
+        <div class="float-left">
+        {{ 'fieldview.content'|trans({}, 'fieldview')|desc('Content') }}
+        </div>
+        <form class="form-inline float-right">
+            <select class="form-control change-location-language">
+                {% for language in languages %}
+                    <option value="{{ path('_ezpublishLocation', {'locationId': location.id, 'language': language.languageCode}) }}"
+                        {% if app.request.get('language') == language.languageCode %} selected="selected" {% endif %}
```

Comment:
How about passing selected language as a parameter to the template instead of accessing the request?